src/templates/layouts/vetCenter/index.tsx

```diff
@@ -0,0 +1,344 @@
+import { VetCenter as FormattedVetCenter } from '@/types/formatted/vetCenter'
+import { GoogleMapsDirections } from '@/templates/common/googleMapsDirections'
+import { Hours } from '@/templates/components/hours'
+import { ImageAndStaticMap } from '@/templates/components/imageAndStaticMap'
+import { AlertBlock } from '@/templates/components/alertBlock'
+import { WysiwygField } from '@/templates/components/wysiwyg'
+import HealthServices from '@/templates/components/healthServices'
+import { FeaturedContent } from '@/templates/common/featuredContent'
+import { QaSection } from '@/templates/components/qaSection'
+export function VetCenter({
+  address,
+  ccNonTraditionalHours,
+  ccVetCenterCallCenter,
+  ccVetCenterFaqs,
+  geolocation,
+  featuredContent,
+  introText,
+  officeHours,
+  officialName,
+  phoneNumber,
+  healthServices,
+  counselingHealthServices,
+  referralHealthServices,
+  otherHealthServices,
+  image,
+  prepareForVisit,
+  title,
+  fieldFacilityLocatorApiId,
+  path,
+}: FormattedVetCenter) {
+  const directionsString = [
+    address?.address_line1,
+    address?.locality,
+    address?.administrative_area,
+  ]
+
+  const structuredSchemaData = {
+    '@context': 'https://schema.org',
+    '@type': 'Place',
+    address: {
+      '@type': 'PostalAddress',
+      streetAddress: `${address.address_line1}${
+        address.address_line2 ? `, ${address.address_line2}` : ''
+      }`,
+      addressLocality: address.locality,
+      addressRegion: address.administrative_area,
+      postalCode: address.postal_code,
+    },
+    name: title,
+    telephone: phoneNumber,
+    openingHoursSpecification: officeHours.map((hours) => ({
+      '@type': 'OpeningHoursSpecification',
+      dayOfWeek: `https://schema.org/${hours.day}`,
+      opens: hours.starthours,
+      closes: hours.endhours,
+    })),
+    hasMap: `https://maps.google.com?saddr=Current+Location&daddr=${encodeURIComponent(
+      `${address.address_line1}, ${address.locality}, ${address.postal_code}`
+    )}`,
+    // "image": [media.entity.image.derivative.url],
```

Comment:
why is this commented out? `image` is passed above, something like `image.links.2_1_large.href` likely has a valid url